test/api/unit/services/uaaStrategy.test.js

```diff
@@ -99,11 +102,14 @@ describe('verifyUAAUser', () => {
 
     cfUAANock.mockVerifyUserGroup(uaaId, uaaUserResponse);
```

Comment:
Do we still need to mock this since we took it out of the `verifyUAAUser`?

Reply:
Good call. We actually aren't even using the `verifyUserGroup` method of the `UAAClient` (outside of one function noted below) so I'm inclined to remove the entire method (and all associated mocks).

The one remaining use is in the `Organization` service method `isUAAAdmin`. This in turn is used in only two other functions: invites to an org and creating orgs. Instead of checking the `UAAIdentity`, I'm going to see if there is an easy way to pass on the user role information that we set via passport